workflow/executor/agent.go

```diff
@@ -133,7 +134,7 @@ func (ae *AgentExecutor) executeHTTPTemplate(ctx context.Context, tmpl wfv1.Temp
 		return nil, err
 	}
 	outputs := &wfv1.Outputs{}
-	outputs.Parameters = append(outputs.Parameters, wfv1.Parameter{Name: "result", Value: wfv1.AnyStringPtr(response)})
+	outputs.Result = pointer.StringPtr(response)
```

Comment:
This is breaking change. it may break existing workflows. Please revert it. We did as a parameter so downline step can refer to it. But now it is too late to change.

Reply:
I've try access HTTP template's output with steps and getting same error `Internal Server Error: templates.main.steps failed to resolve {{steps.good.outputs.parameters.result}}`, in v3.2.3 and v3.2.4. So looks like no one can use it's output in downline step currently. So this isn't a breaking change I think.

I rarely use steps, if I tested it in wrong way pls forgive me.

Reply:
got it